airflow/providers/google/facebook_ads_to_gcs/example_dags/example_ads.py

```diff
@@ -0,0 +1,110 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""
+Example Airflow DAG that shows how to use FacebookAdsReportToGcsOperator.
+"""
+from facebook_business.adobjects.adsinsights import AdsInsights
+
+from airflow import models
+from airflow.providers.google.cloud.operators.bigquery import (
+    BigQueryCreateEmptyDatasetOperator, BigQueryCreateEmptyTableOperator, BigQueryExecuteQueryOperator,
+)
+from airflow.providers.google.cloud.operators.gcs_to_bigquery import GCSToBigQueryOperator
+from airflow.providers.google.facebook_ads_to_gcs.operators.ads import FacebookAdsReportToGcsOperator
+from airflow.utils.dates import days_ago
+
+# [START howto_GCS_env_variables]
+GCP_PROJECT_ID = "free-tier-1997"
+GCS_BUCKET = "airflow_bucket_fb"
+GCS_OBJ_PATH = "Temp/this_is_my_report_csv.csv"
+GCS_CONN_ID = "google_cloud_default"
+DATASET_NAME = "airflow_test_dataset"
+TABLE_NAME = "airflow_test_datatable"
+# [END howto_GCS_env_variables]
+
+# [START howto_FB_ADS_env_variables]
+FACEBOOK_ADS_CONN_ID = "facebook_default"
```

Comment:
Can you retrieve these values from environment variables? This will show that you should not store those values here.
```
TABLE_NAME = os.environ.get("FB_TABLE_NAME", "airflow_test_datatable")
```

Also I would remove the `FACEBOOK_ADS_CONN_ID` to present fallback to default connection